selfdrive/car/ford/values.py

```diff
@@ -1,21 +1,98 @@
+from collections import namedtuple
+from dataclasses import dataclass
+from enum import Enum
 from typing import Dict, List, Union
 
 from cereal import car
 from selfdrive.car import dbc_dict
-from selfdrive.car.docs_definitions import CarInfo
+from selfdrive.car.docs_definitions import CarFootnote, CarInfo, Column
+
 Ecu = car.CarParams.Ecu
+AngleRateLimit = namedtuple('AngleRateLimit', ['speed_points', 'max_angle_diff_points'])
+
+
+class CarControllerParams:
+  # Messages: Lane_Assist_Data1, LateralMotionControl
+  LKAS_STEER_STEP = 5
+  # Message: IPMA_Data
+  LKAS_UI_STEP = 100
+  # Message: ACCDATA_3
+  ACC_UI_STEP = 5
+
+  STEER_RATE_LIMIT_UP = AngleRateLimit(speed_points=[0., 5., 15.], max_angle_diff_points=[5., .8, .15])
+  STEER_RATE_LIMIT_DOWN = AngleRateLimit(speed_points=[0., 5., 15.], max_angle_diff_points=[5., 3.5, 0.4])
 
-MAX_ANGLE = 87.  # make sure we never command the extremes (0xfff) which cause latching fault
+
+class CANBUS:
+  main = 0
+  camera = 2
 
 
 class CAR:
-  FUSION = "FORD FUSION 2018"
+  ESCAPE_MK4 = "FORD ESCAPE 4TH GEN"
+  FOCUS_MK4 = "FORD FOCUS 4TH GEN"
+
+
+class Footnote(Enum):
```

Comment:
no need for any of this docs stuff yet. we'll add once they're out of dashcam only